content/en/docs/HOWTOs/EventFeeds/TranslationHowTo.md

```diff
@@ -0,0 +1,638 @@
+---
+title: "Writing an XSLT Translation"
+linkTitle: "Writing a Translation"
+#weight:
+date: 2022-06-07
+tags:
+  - processing
+  - feed
+description: >
+  This HOWTO will take you through the production of an XSLT for a feed, including issues such as event filtering, common errors and testing.
+---
+
+<!-- Created with Stroom v6.1-beta.16 -->
+
+{{% see-also %}}
+[HOWTO - Event Processing]({{< relref "ProcessingHowTo" >}})
+{{% /see-also %}}
+
+
+## Introduction
+
+This document is intended to explain how and why to produce a translation within stroom and how the translation fits into the overall processing within stroom.
+It's not intended as an XSLT tutorial so a basic XSLT knowledge must be assumed.
+The document will contain potentially useful XSLT fragments to show how certain processing activities can be carried out.
+As with most programming languages, there are likely to be multiple ways of producing the same end result with different degrees of complexity and efficiency.
+Examples here may not be the best for all situations but do reflect experience built up from many previous translation jobs.
+
+The document should be read in conjunction with other online stroom documentation, in particular [Event Processing]({{< relref "ProcessingHowTo" >}}).
+
+
+## Translation Overview
+
+The translation process for raw logs is a multi-stage process defined by the processing pipeline:
+
+### Parser
+
+The parser takes raw data and converts it into an intermediate XML document format.
```

Comment:
Add glossary link for parser and any other common terms.
https://gchq.github.io/stroom-docs/7.0/community/documentation/style-guide/additional-elements/#glossary-links

Reply:
Unable ot do so currently as the legacy branch has no glossary.